docs/whitepaper/federated_sidetree.md

```diff
@@ -1,25 +1,27 @@
 # Multitree: Federated Sidetree DID method
 
 Status: concept whitepaper
-Version: Dec 5, 2020
+Version: Dec 7, 2020
 
 ## Introduction
 
 Multitree* is a [decentralized identifier](https://www.w3.org/TR/did-core/) (DID) method that enables trusted DID verification over a federation of DID publishers. The method is loosely modelled on the concept of a [Fediverse](https://en.wikipedia.org/wiki/Fediverse), where an operator of a verifiable registry can start with their own instance, and also be able to interconnect with other verifiable registry operators over time. DID controllers may migrate their DIDs over different DID publishers over time. We also introduce the concept of a Multitree Hub, where a group of DID observers/witnesses support resolution over multiple DID registry operators.
 
+Multitree is designed to support interoperability across independent registries and be agnostic to implementation choices within these registries. Each independent operator can form their own "Multitree" registry and these registries can be interconnected. We use a data model is independent of a protocol, database or DLT. In particular, we enable interconnected independent registries without up-front coordination nor a dependency on using the same registry implementation. These registries can independently form over time and then choose to interconnect.
```

Comment:
We use a data model --that is-- independent ...

Reply:
@bstasyszyn fixed.